bemani/utils/read.py

```diff
@@ -1502,11 +1750,16 @@ def __gather_sound_files(self, directory: str) -> Dict[int, str]:
             for filename in filenames:
                 songid, extension = os.path.splitext(filename)
                 if extension == '.1' or extension == '.ifs':
-                    try:
-                        files[int(songid)] = os.path.join(directory, os.path.join(dirpath, filename))
-                    except ValueError:
-                        # Invalid file
-                        pass
+                    if filename != '12030-p0.ifs':
```

Comment:
Haha what bullshit is this working around now? Maybe a comment about what's going on...

Reply:
A friend of mine told me specifically to not extract this one because for some reason the leggendaria charts for POODLE were already stored in the actual chart ifs file. And this overall essentially checks if there's an add on ifs file and extracts that one instead. I'll add a comment though yeah